### PR TITLE
Add stubs for use by entry point

### PR DESCRIPTION
**Pull request by shelbyspees**

### CODE REVIEW

File: lib/revolution/build.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+module Build
+  def build_package
```
Comment (smaslennikov) [inline lib/revolution/build.rb]: This should probably have some arguments to pull in (such as package name and location)

Comment (smaslennikov) [reply lib/revolution/build.rb]: Forget it, this is stubby enough.

Comment (shelbyspees) [reply lib/revolution/build.rb]: I added arguments, updating soon

File: lib/revolution/build.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+module Build
+  def self.build_package(name = 'test', location = 'Los Angeles, CA')
```
Comment (smaslennikov) [inline lib/revolution/build.rb]: This isn't a meaningful location we'd ever use. This should be a path

**Pull request merged**